### PR TITLE
Minor fixes after removal of key auto-generation (replacement of #2384)

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Do minor fixes after removal of key auto-generation (replacement of #2384).

- at build time, in case the default ledger is not in the available wallet addresses, raise an error.

## Fixes

replaces #2384 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

n/a

### CODE REVIEW

File: aea/cli/run.py [hunk 1]
```diff
@@ -195,5 +197,14 @@ def _build_aea(
         )
         aea = builder.build(connection_ids=connection_ids, password=password)
         return aea
+    except AEAWalletNoAddressException:
+        error_msg = (
+            "You haven't specified any private key for the AEA project.\n"
+            "Please add one by using the following commands:\n"
+            "\n"
+            f"    aea generate-key {DEFAULT_LEDGER}\n"
+            f"    aea add-key fetchai {PRIVATE_KEY_PATH_SCHEMA.format(DEFAULT_LEDGER)}\n"
```
Comment (DavidMinarsch) [inline aea/cli/run.py:206]: Hm, not sure it makes sense to tie this to default. Why not just say:
`Please add one by using the command `aea genera-key` and `aea add-key` for the ledger of your choice.`?

Comment (marcofavorito) [reply aea/cli/run.py:206]: ok

Comment (DavidMinarsch) [inline aea/cli/run.py:206]:
```suggestion
            "Please add one by using the commands `aea generate-key` and `aea add-key` for the ledger of your choice.\n"
```

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**